src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -169,6 +169,9 @@ def evaluate_value(self, value: T) -> DataConditionResult:
         result = handler.evaluate_value(value, self.comparison)
         return self.get_condition_result() if result else None
 
+    def get_preview_groups(self, group_ids: set[int]) -> set[int]:
```

Comment:
placeholder, should implement for every `DataCondition` `type`, via the `condition_handler_registry`?